package.json

```diff
@@ -42,9 +42,9 @@
     "html4-id": "^1.0.0",
     "loader-utils": "^1.0.4",
     "svg4everybody": "^2.1.0",
-    "svgo": "^0.6.6",
+    "svgo": "^1.0.3",
```

Comment:
Could you look into making this a `peerDependency`? That might work better as you're free to use whichever version you'd like.